README.md

```diff
@@ -16,8 +16,45 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using `.click()`, `.change()`, et al as event handlers]
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Avoid using `.click()`, `.change()`, et al as event handlers
```

Comment:
Add summary explaining jQuery supports both `.on('eventName')` and aliases `.eventName()`? Link to the docs.